src/components/dev-hub/input.js

```diff
@@ -38,22 +25,16 @@ const StyledInput = styled('input')`
     padding: ${({ narrow }) =>
         narrow ? `${size.small} ${size.medium}` : size.medium};
     position: relative;
-    width: calc(
```

Comment:
@jestapinski, do you remember the reason for this width? It was causing some weird spacing issues on the modal and it seems to look fine without it but I want to make sure I'm not unknowingly breaking something.

Reply:
I think it had do to with the border but also some padding? I remember using that to line the input up wiht the correct width in storybook, but that also seems a bit short now! If it looks fine without it let's go ahead and remove it. Thanks!